doc/texdoc.tex

```diff
@@ -787,13 +787,7 @@ \subsection{Zsh}
 \item adding \code{eval "\$(texdoc \lopt{print-completion} zsh)"} to your
   zsh configuration file.
 
-\item adding a file somewhere in \code{\$fpath} with the following contents:
-%
-\begin{htcode}
-#compdef texdoc
-(( $+functions[__texdoc] )) || eval "$(texdoc \textbackslash
-  --print-completion zsh)" && __texdoc
-\end{htcode}
+\item creating a file named \code{_texdoc} in \code{\$fpath} whose content is the output of \code{texdoc \lopt{print-completion} zsh}.
```

Comment:
We need an escape backslash for `_` here, so `\code{\_texdoc}`, otherwise the compilation of this document has failed

Reply:
Fixed.